docs/administrator-handbook.md

```diff
@@ -137,6 +132,51 @@ changes from the pending deployment to the booted deployment.
 
 See `man rpm-ostree` for more information.
 
+## Using overrides and `usroverlay`
+
+While some people talk about "immutability" when referring to image-based
+systems like rpm-ostree, in fact a top level goal of rpm-ostree is
+to *empower* users and system administrators.  When something goes
+wrong, you are root on your own computer and should have the ability to apply
+overrides locally.
+
+First, there is the `rpm-ostree override replace` command, which
+will replace an RPM, and apply that change persistently for the *next*
+boot - this is symmetric with how `rpm-ostree install` works.
+
+For example, suppose you want to test a fix to `podman`.  You can pass
+both direct HTTP URLs as well as local files:
+
+```
+$ rpm-ostree override replace https://kojipkgs.fedoraproject.org//packages/podman/3.3.1/1.fc34/x86_64/podman-3.3.1-1.fc34.x86_64.rpm
+```
+
+Now, suppose that you want to test this change *live*.  There are two choices.
+The first choice is to run
```

Comment:
This choice assumes that you did `rpm-ostree override replace` as above, while the second choice doesn't require this. Maybe let's make that clearer by saying e.g.

```suggestion
The first choice is to run the `rpm-ostree override replace` command above to stage the deployment, and then run
```